src/pages/ExamsPage/components/AllowanceList.jsx

```diff
@@ -39,6 +39,7 @@ const AllowanceList = () => {
       examName,
       allowanceType: formatMessage(messages.allowanceTypeMinutes),
       extraTimeMins,
+      username,
```

Comment:
I've only added the username property -- is it better to have all of the properties included? I wasn't sure if there was a reason we were limiting to begin with.

Reply:
Done!